### ISSUE TITLE
Pagination accessibility focus not working

### ISSUE THREAD

**Issue by fredriklindell** [swimlane/ngx-datatable#1684]
**I'm submitting a ...**
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
Pager pages li element not focused/highlighted if using keyboard navigation (tab) to get there.

**Expected behavior**
Focused/Highlighted pager page should be outlined with a border for accessibility.

**Reproduction of the problem**
Please see this exmaple: http://swimlane.github.io/ngx-datatable/#virtual-paging, a user is able to tab to a page number and then press enter to go there. However the page number container is not focused/highlighted.

**What is the motivation / use case for changing the behavior?**
This is needed from an Accessibility point of view.

**Please tell us about your environment:**
Windows 10

* **Table version:** 
14.0.0

* **Angular version:** 
Angular 7.1.3

* **Browser:**
all

* **Language:** 
TypeScript 3.1.6

**Comment by nSedrickm** [comment]
Hi, I am currently working on improving the accessibility of ceph-dashboard [datatable component](https://github.com/ceph/ceph/pull/46958) which is also affected by this issue. The pagination elements are not focusable. I would like to work on a fix
